cmd/antrea-agent/agent.go

```diff
@@ -337,6 +327,36 @@ func run(o *Options) error {
 		return err
 	}
 
+	var flowExporter *exporter.FlowExporter
+	if features.DefaultFeatureGate.Enabled(features.FlowExporter) {
+		flowExporterOptions := &flowexporter.FlowExporterOptions{
+			FlowCollectorAddr:      o.flowCollectorAddr,
+			FlowCollectorProto:     o.flowCollectorProto,
+			ActiveFlowTimeout:      o.activeFlowTimeout,
+			IdleFlowTimeout:        o.idleFlowTimeout,
+			StaleConnectionTimeout: o.staleConnectionTimeout,
+			PollInterval:           o.pollInterval}
+		flowExporter, err = exporter.NewFlowExporter(
+			ifaceStore,
+			proxier,
+			config.IsIPv4Enabled(nodeConfig, networkConfig.TrafficEncapMode),
+			config.IsIPv6Enabled(nodeConfig, networkConfig.TrafficEncapMode),
+			k8sClient,
+			nodeRouteController,
+			networkConfig.TrafficEncapMode.IsNetworkPolicyOnly(),
```

Comment:
could we pass `networkConfig` as a parameter and get rid of `config.IsIPv4Enabled(nodeConfig, networkConfig.TrafficEncapMode)`, `config.IsIPv6Enabled(nodeConfig, networkConfig.TrafficEncapMode)` and `networkConfig.TrafficEncapMode.IsNetworkPolicyOnly()` as a consequence. You can also pass `networkConfig. TrafficEncapMode` instead of the whole `networkConfig`.

Reply:
done